Tests/TracingInstrumentationTests/SpanTests.swift

```diff
@@ -128,9 +129,11 @@ final class SpanTests: XCTestCase {
         XCTAssertEqual(attributes.name, SpanAttribute.string("kappa"))
         XCTAssertEqual(attributes.name, "kappa")
         XCTAssertEqual(attributes.sampleHttp.statusCode, 200)
+        #endif
```

Comment:
```
#else 
print some info about shy this is skipped?
#endif
```

Reply:
What do you mean by "print"? Something like `#warning` or an actual runtime `print`?

Reply:
Added a comment for it.

Reply:
I meant `print("oh oh")` to be honest :) 
In my own test suites I usually print something in yellow for such tests...

but either's fine really